src/components/definition-list/dd.component.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledDd } from './definition-list.style';
+
+const Dd = ({
+  mb = 2,
+  content = 'Content for associated label'
+}) => {
+  return (
+
+    <StyledDd
+      mb={ mb }
+    >
+      {content}
+    </StyledDd>
+  );
+};
+
+Dd.propTypes = {
+  /** Content margin bottom. This value will be multiplied by the theme spacing constant (8) */
+  mb: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6]),
+  /** Prop for what will render in the `<Dd></Dd>` tags */
+  content: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
```

Comment:
I think this prop should just be `children` not `content` then you can just use this component in the normal way, i.e.
`<Dd><ChildElements/></Dd>`

Reply:
✅